### ISSUE TITLE
`undefined method `find_all' for Gem::Specification:Class` when trying to connect

### ISSUE THREAD

**Issue by peterbe** [bobthecow/genghis#81]
http://cl.ly/N9IE

https://gist.github.com/peterbe/5032702

**Comment by bobthecow** [comment]
Hrm.

What versions of Ruby and Gem are you running?

**Comment by peterbe** [comment]
```
peterbe@host-6-53:~$ ruby --version
ruby 1.8.7 (2012-02-08 patchlevel 358) [universal-darwin12.0]
peterbe@host-6-53:~$ gem --version
1.3.6
```

**Comment by bobthecow** [comment]
my, that is an old gem version you have :)

```
$ gem --version
1.8.23
```

**Comment by bobthecow** [comment]
That message, while annoying, doesn't actually hurt anything. It's trying to check the currently installed versions of `genghisapp` and `bson_ext` gems so it can show you some friendly status messages.

I'll get a fix for that into the next release, but in the meantime, `gem update --system`.

**Comment by peterbe** [comment]
Awesome! Upgrading gem worked. I'm in. 

I guess the issue is that the docs should either mention to use modern versions of gem or it should warn you if you install with an old version. Or something like that. Your call.

**Comment by bobthecow** [comment]
This has been fixed in `v2.3.0`. Thanks for reporting it :)
